src/Microsoft.Azure.SignalR/HubHost/ServiceConnection.cs

```diff
@@ -241,20 +249,39 @@ internal Task StopAsync()
             }
         }
 
-        private Timer StartTimeoutTimer()
+        private Timer StartKeepAliveTimer()
```

Comment:
Since we're moving to a single timer model here I would change this logic to use the TimerAwaitable.

Reply:
Sure. Let me update it.